pkgs/applications/misc/nemu/default.nix

```
@@ -0,0 +1,117 @@
+{ stdenv, config, lib, fetchFromGitHub
+, cmake
+, pkgconfig
+, gettext
+, libpthreadstubs
+, libudev
+, libusb1
+, ncurses
+, qemu
+, sqlite
+
+, libxml2 ? null
+, libarchive ? null
+, graphviz ? null
+, dbus ? null
```

```suggestion
, libxml2
, libarchive
, graphviz
, dbus
```

This is not required. I would appreciate it if you could move the packages to the options they belong. Ideally below or behind them.